chall05/vsempere.c

```diff
@@ -0,0 +1,61 @@
+# include <stdlib.h>
+
+static size_t		ft_strlen(const char *s)
+{
+	size_t	i;
+
+	i = 0;
+	while (s[i] != 0)
+		i++;
+	return (i);
+}
+
+static char		*ft_strstr(const char *big, const char *little)
+{
+	size_t		i;
+	size_t		j;
+
+	i = 0;
+	j = 0;
+	while (big[i])
+	{
+		while (big[i + j] == little[j] && big[i + j])
+			j++;
+		if (little[j] == '\0')
+			return ((char *)(big + i));
+		j = 0;
+		i++;
+	}
+	return (NULL);
+}
```

Comment:
You could use the standard library, in those challs, you are encouraged to take advantage of the whole standard library, you already have the cursus' projects to learn about how they work 😉